demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime-http.yaml

```diff
@@ -1,35 +0,0 @@
-apiVersion: serving.kserve.io/v1alpha1
```

Comment:
This duplicates https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-tgis/caikit-tgis-servingruntime.yaml